Content.Shared/Jobs/JobPrototype.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Utility;
+using YamlDotNet.RepresentationModel;
+
+namespace Content.Shared.Jobs
+{
+    [Prototype("job")]
+    public class JobPrototype : IPrototype, IIndexedPrototype
+    {
+        public string ID { get; private set; }
+        public string Name { get; private set; }
+
+        public string StartingGear { get; private set; }
+        public IEnumerable<string> Department { get; private set; }
+
+        public void LoadFrom(YamlMappingNode mapping)
+        {
+            ID = mapping.GetNode("id").AsString();
+            Name = mapping.GetNode("name").ToString();
+            StartingGear = mapping.GetNode("startinggear").ToString();
```

Comment:
I think this yaml node name would be more readable as `startingGear` or `StartingGear`.

Reply:
It should be camelCase `startingGear` based on what we have for the other YAML nodes.

Reply:
You are right. I choose this because I opened a random yaml file and this was the syntax used. Seemed strange to me but I did the same. But now that I open other yaml file I see the your version (which is better).

This was the one. https://github.com/space-wizards/space-station-14/blob/master/Resources/Prototypes/Technologies/technologies.yml